buildpacks/builder.go

```diff
@@ -39,6 +39,7 @@ var (
 		"gcr.io/paketo-buildpacks",
 		"docker.io/paketobuildpacks",
 		"ghcr.io/vmware-tanzu/function-buildpacks-for-knative",
+		"gcr.io/buildpacks/builder"
```

Comment:
This should not be the full image name. Just use the image registry and prefix. With [this change](https://github.com/knative/func/pull/1442), the prefix should also include a slash at the end.
```suggestion
		"gcr.io/buildpacks/"
```